static/js/components/email/EmailCompositionDialog_test.js

```diff
@@ -94,6 +100,24 @@ describe('EmailCompositionDialog', () => {
     assert.isTrue(sendStub.called, "called send handler");
   });
 
+  it('should rename "send" button to "Save Changes"', () => {
```

Comment:
Please change the description to `'should show a "Save" label when the dialog is being used to edit an email'`